src/storage/mvcc/reader/reader.rs

```diff
@@ -95,6 +95,11 @@ impl<S: EngineSnapshot> SnapshotReader<S> {
         self.reader.seek_write(key, ts)
     }
 
+    #[inline(always)]
+    pub fn get_value(&mut self, key: &Key, start_ts: TimeStamp) -> Result<Option<Value>> {
```

Comment:
`SnapshotReader` typically uses its own `start_ts` for something. If it doesn't need the `start_ts` of `SnapshotReader` itself, I suggest to just use the inner `MvccReader` directly.

Reply:
Removed. PTAL.

BTW, your suggestion made me realize that there is a mix of `SnapshotReader` and `MvccReader` in the flashback commands, and I think we can open another separate PR to improve this situation, after this PR is merged, of course.

Reply:
address this comment at https://github.com/tikv/tikv/pull/13860 @sticnarf @JmPotato